example/test_problem/Hydro/SinkParTest/Analysis.py

```diff
@@ -0,0 +1,186 @@
+#**************************************************************************#
+# Code:        GetColMap                                                   # 
+#                                                                          #
+# Author:      Wei-An Chen                                                 #
+#                                                                          #
+# This scipt is useful for checking the evolution of simulation by         #
+# plotting the column density map.                                         #
+#                                                                          #
+# s_index:   the starting index of the snapshot,                           #
+#            e.g. s_index = 10 for Data_000010.                            #
+# e_index:   the end index of the snapshot.                                #
+# npixel:    pixel number of the map at each dimension.                    #
+# ProjPlane: the projection plane [xy/xz].                                 #
+#**************************************************************************#
+
+import yt
+from astropy.constants import m_p
+import numpy as np
+import os
+import matplotlib.pyplot as plt
+import matplotlib.cm as cm
+from matplotlib import colors
+import matplotlib as mpl
+
+mpl.rcParams.update({'font.size': 15})
+mpl.rcParams['axes.linewidth'] = 2
+mpl.rcParams['patch.linewidth'] = 2
+mpl.rcParams['xtick.major.width'] = 2
+mpl.rcParams['xtick.minor.width'] = 2
+mpl.rcParams['ytick.major.width'] = 2
+mpl.rcParams['ytick.minor.width'] = 2
+
+s_index   = 0
+e_index   = 300
```

Comment:
I think it will be better to use a similar interface as other test problems, e.g., https://github.com/gamer-project/gamer/blob/main/example/test_problem/Hydro/Plummer/plot_script/plot_gas.py#L5-L29